setup.py

```diff
@@ -66,6 +66,7 @@
         "pydantic>=1.8.2,<2.0.0",
         "eth-utils>=1.10.0,<3.0",
         "py-cid>=0.3.0,<0.4.0",
+        "pysha3>=1.0.2,<2.0.0",
```

Comment:
Is this used anywhere? For some reason I am not seeing it

Reply:
`eth-hash` backend